includes/hardlinks/breakdown-folder-structure-docker.md

```diff
@@ -4,50 +4,90 @@
 
 qBittorrent, Deluge, ruTorrent
 
-The reason why we use `/data/torrents` for the torrent client is because it only needs access to the torrent files. In the torrent software settings, you’ll need to reconfigure paths and you can sort into sub-folders like `/data/torrents/{tv|movies|music}`.
+The reason we use `/data/torrents` for the torrent client is because it only needs access to the torrent files. In the torrent software settings, you’ll need to reconfigure paths and can then sort into subfolders like `/data/torrents/{tv|movies|music}`.
 
 {! include-markdown "docker-tree-torrents.md" !}
 
 `Container Path:` => `/data/torrents/`
 
 `Host Path:` => `/<path_to_data>/data/torrents/`
 
+Docker-compose volumes example:
+
+```none
+[...]
+    volumes:
+	  [...]
+      - /<path_to_data>/data/torrents:/data/torrents
+[...]
+```
+
 #### Usenet clients
 
 NZBGet or SABnzbd
 
-The reason why we use `/data/usenet` for the usenet client is because it only needs access to the usenet files. In the usenet software settings, you’ll need to reconfigure paths and you can sort into sub-folders like `/data/usenet/complete/{tv|movies|music}`.
+The reason we use `/data/usenet` for the usenet client is because it only needs access to the usenet files. In the usenet software settings, you’ll need to reconfigure paths and can then sort into subfolders like `/data/usenet/complete/{tv|movies|music}`.
 
 {! include-markdown "docker-tree-usenet.md" !}
 
 `Container Path:` => `/data/usenet/`
 
 `Host Path:` => `/<path_to_data>/data/usenet/`
 
+Docker-compose volumes example:
+
+```none
+[...]
+    volumes:
+	  [...]
+      - /<path_to_data>/data/usenet:/data/usenet
+[...]
+```
+
 #### The Starr Apps
 
 Sonarr, Radarr, Readarr and Lidarr
 
-Sonarr, Radarr, Readarr and Lidarr gets access to everything using `/data` because the download folder(s) and media folder will look like and be one file system. Hardlinks will work and moves will be atomic, instead of copy + delete.
+Sonarr, Radarr, Readarr and Lidarr get access to everything in `/data` because the download and media folder(s) will look like and will be one file system. Hardlinks will work and moves will be atomic (*instead of copy + delete*).
 
 {! include-markdown "docker-tree-full.md" !}
 
 `Container Path:` => `/data`
 
 `Host Path:` => `/<path_to_data>/data/`
 
+Docker-compose volumes example:
```

Comment:
```suggestion
Abbreviated Docker-compose volumes example
```

Reply:
> 

I think the word `Abbreviated` is to complicated for the non native English speaking people.

Reply:
How about "Docker-compose Volumes Example Snippet" ?

Reply:
Or Example Docker-compose Volumes Excerpt?